### ISSUE TITLE
Clicking on the podStation button is not bringing existing tabs into focus

### ISSUE THREAD

**Issue by dellagustin** [podStation/podStation#310]
Clicking on the podStation button used to either:

- open a new podStation tab, if no podStation tab is already open OR
- bring a podStation tab into focus, if a podStation tab exists

In later versions (current 1.47.1), it is always opening a new tab, even though there was no related code change for that.

## Related code points and changes
- https://github.com/podStation/podStation/pull/60
- https://github.com/podStation/podStation/blob/b8588905fca379523863b10e00ba7b65b71af694/src/background/podstation_bg.js#L4-L28

## References

- https://groups.google.com/a/chromium.org/g/chromium-extensions/c/sgtyJxodj7k
- https://developer.chrome.com/docs/extensions/reference/extension/#method-getViews
- https://bugs.chromium.org/p/chromium/issues/detail?id=1045813#c2

**Comment by dellagustin** [comment]
`chrome.extension.getViews({type:"tab"})` is returning an empty set.
I'm waiting for replies to https://groups.google.com/a/chromium.org/g/chromium-extensions/c/sgtyJxodj7k.

**Comment by dellagustin** [comment]
I found something that gave me a hint:
- https://www.tutorialguruji.com/javascript/chrome-extension-getviews-and-extension-options-page/
- https://bugs.chromium.org/p/chromium/issues/detail?id=341459
An old issue that stated that tabs open by window.open were not returned by `chrome.extension.getViews({type:"tab"})`. This old issue is already marked as closed, but I verified that this is what is happened now with podStation.
I created a new issue: https://bugs.chromium.org/p/chromium/issues/detail?id=1298195
A workaround is to use `chrome.tabs.create`.
